contracts/validator-manager/PoSValidatorManager.sol

```diff
@@ -165,13 +174,31 @@ abstract contract PoSValidatorManager is
         bool includeUptimeProof,
         uint32 messageIndex
     ) external {
+        if (!_initializeEndPoSValidation(validationID, includeUptimeProof, messageIndex)) {
+            revert ValidatorIneligibleForRewards();
+        }
+    }
+
+    function forceInitializeEndValidation(
```

Comment:
can youy add natspec comments

Reply:
Natspec comments are included in the interface definition.

Reply:
generally still have a {See Interface-method} comment for functions of an interface

Reply:
Wow we're very inconsistent with natspec on implementation functions. I'm going to defer this to a generalized cleanup ticket.